lib/jekyll/site.rb

```diff
@@ -73,10 +73,11 @@ def process
       render
       cleanup
       write
-      print_stats
+      print_stats if @config["profile"]
     end
 
     def print_stats
+      # TODO: remove duplicate guard statement in next major version change
       if @config["profile"]
```

Comment:
If you're to move the `if` above, then just remove it here. That means that this method will always print stats if called.